doc/examples/qubitekk/ex_qubitekkcc.py

```diff
@@ -7,7 +7,8 @@
 
 def main():
     if _platform == "linux" or _platform == "linux2":
-        cc1 = ik.qubitekk.CC1v2001.open_serial('/dev/ttyUSB0', 19200, timeout=1)
+        cc1 = ik.qubitekk.CC1v2001.open_serial('/dev/ftdi_xxxx', 19200,
```

Comment:
same as before

also lets say `baud=19200` so that its clear for the example to the reader